changes/ce/feat-12201.en.md

```diff
@@ -0,0 +1 @@
+Support hot update of TCP/SSL/WS/WSS MQTT listeners configuration, which allows changing most of the configuration parameters without restarting the listener and disconnecting the clients.
```

Comment:
Maybe list the ones which are NOT supported at the moment.

The frontend popup dialog when updating listener config should be changed to something like:
"Updating abc and xyz will need to restart listener".

Reply:
also add "if update fail, listener will be restarted and lost all the connections"